### ISSUE TITLE
Get a not-implemented popup when you attempt to rename a resource in RESX

### ISSUE THREAD

**Issue by nguerrera** [dotnet/roslyn-project-system#752]
Create a .net core project, add a resx, make some changes in the editor:

```
---------------------------
Microsoft Visual Studio
---------------------------
The method or operation is not implemented.
---------------------------
OK   
---------------------------
```

The edit actually succeeds after you dismiss the popup.

Possibly another issue with same root cause as #747 and #686 

@333fred Can you check this too when fixing #686

**Comment by 333fred** [comment]
I'm unable to replicate this @nguerrera. Are you talking about the resx editor, or some other editor?

**Comment by nguerrera** [comment]
Hmm, seems its not so deterministic. This worked for me twice in a row:

1. File -> New project
2. Console App (.NET Core)
3. Right click, add new item -> Resources file
4. Rename String1 to String2

**Comment by nguerrera** [comment]
@333fred Actually, it seems renaming a key is the trigger and it's probably deterministic.

**Comment by 333fred** [comment]
Ah, there we go. Thanks @nguerrera.

**Comment by 333fred** [comment]
```
Microsoft.VisualStudio.LanguageServices.dll!Microsoft.VisualStudio.LanguageServices.Implementation.Utilities.Exceptions.ThrowENotImpl() Line 32	C#
 Microsoft.VisualStudio.LanguageServices.Implementation.dll!Microsoft.VisualStudio.LanguageServices.Implementation.CodeModel.CodeModelProjectCache.GetOrCreateFileCodeModel(string filePath, object parent) Line 135	C#
Microsoft.VisualStudio.LanguageServices.Implementation.dll!Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.CPS.CPSProjectCodeModel.GetFileCodeModel(EnvDTE.ProjectItem item) Line 75	C#
Microsoft.VisualStudio.LanguageServices.Implementation.dll!Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.CPS.CPSProject.GetFileCodeModel(EnvDTE.ProjectItem item) Line 44	C#
Microsoft.VisualStudio.LanguageServices.Implementation.dll!Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.CPS.CPSCodeModelFactory.GetFileCodeModel(Microsoft.VisualStudio.LanguageServices.ProjectSystem.IWorkspaceProjectContext context, EnvDTE.ProjectItem item) Line 18	C#
Microsoft.VisualStudio.ProjectSystem.Managed.VS.dll!Microsoft.VisualStudio.ProjectSystem.VS.LanguageServices.ProjectContextCodeModelProvider.GetFileCodeModel(EnvDTE.ProjectItem fileItem) Line 50	C#
```

Looks like a language service assert. I can also hit it without doing a rename, it's just not popping up a dialog. The fileItem being passed into the GetFileCodeModel call is a OAProjectItem for the resource file.

**Comment by 333fred** [comment]
Summoning @davkean, now that I can.

**Comment by davkean** [comment]
No idea what its trying to do, would need to step through it.

**Comment by srivatsn** [comment]
Fixed by #1163
